web_src/js/features/notification.js

```diff
@@ -19,21 +19,53 @@ export function initNotificationsTable() {
 }
 
 export function initNotificationCount() {
-  if (NotificationSettings.MinTimeout <= 0) {
+  const notificationCount = $('.notification_count');
+
+  if (!notificationCount.length) {
     return;
   }
 
-  const notificationCount = $('.notification_count');
-
-  if (notificationCount.length > 0) {
-    const fn = (timeout, lastCount) => {
-      setTimeout(async () => {
-        await updateNotificationCountWithCallback(fn, timeout, lastCount);
-      }, timeout);
-    };
+  if (NotificationSettings.EventSourceUpdateTime > 0 && !!window.EventSource) {
+    // Try to connect to the event source first
+    const source = new EventSource(`${AppSubUrl}/user/events`);
```

Comment:
From the 2nd link you've provided I read that the browser will keep retrying after each time the stream is broken. Shouldn't we react with `source.close()` if there is a logoff event?

Reply:
They'll get a 404 and the source will be closed.

Reply:
we do anyway.

Reply:
(see line 52)